deployment/GI01SUMO_Deploy.csv

```diff
@@ -301,3 +301,53 @@ AR35-05,,,,GI01SUMO-SBD12-04-PCO2AA000,6,1,2019-08-05T15:29:00,,CGMGI-01SUMO-000
+AR46,,,,GI01SUMO-RII11-02-CTDMOQ015,7,1,,,CGMGI-01SUMO-00007,,CGINS-CTDMOQ-12937,,,,250,,
+AR46,,,,GI01SUMO-RII11-02-CTDMOQ016,7,1,,,CGMGI-01SUMO-00007,,CGINS-CTDMOQ-12938,,,,350,,
+AR46,,,,GI01SUMO-RII11-02-CTDMOQ017,7,1,,,CGMGI-01SUMO-00007,,CGINS-CTDMOQ-12939,,,,500,,
+AR46,,,,GI01SUMO-RII11-02-CTDMOR018,7,1,,,CGMGI-01SUMO-00007,,CGINS-CTDMOR-13628,,,,750,,
+AR46,,,,GI01SUMO-RII11-02-CTDMOR019,7,1,,,CGMGI-01SUMO-00007,,CGINS-CTDMOR-13629,,,,1000,,
+AR46,,,,GI01SUMO-RII11-02-CTDMOR020,7,1,,,CGMGI-01SUMO-00007,,CGINS-CTDMOR-13630,,,,1500,,
+AR46,,,,GI01SUMO-RII11-02-DOSTAD031,7,1,,,CGMGI-01SUMO-00007,,CGINS-DOSTAD-00516,,,,40,,
+AR46,,,,GI01SUMO-RII11-02-DOSTAD032,7,1,,,CGMGI-01SUMO-00007,,CGINS-DOSTAD-00517,,,,80,,
+AR46,,,,GI01SUMO-RII11-02-DOSTAD033,7,1,,,CGMGI-01SUMO-00007,,CGINS-DOSTAD-00518,,,,130,,
+AR46,,,,GI01SUMO-RII11-02-FLORDG031,7,1,,,CGMGI-01SUMO-00007,,CGINS-FLORDG-04032,,,,40,,
+AR46,,,,GI01SUMO-RII11-02-FLORDG032,7,1,,,CGMGI-01SUMO-00007,,CGINS-FLORDG-04033,,,,80,,
+AR46,,,,GI01SUMO-RII11-02-FLORDG033,7,1,,,CGMGI-01SUMO-00007,,CGINS-FLORDG-04034,,,,130,,
+AR46,,,,GI01SUMO-RII11-02-PCO2WC051,7,1,,,CGMGI-01SUMO-00007,,CGINS-PCO2WC-C0103,,,,40,,
+AR46,,,,GI01SUMO-RII11-02-PCO2WC052,7,1,,,CGMGI-01SUMO-00007,,CGINS-PCO2WC-C0104,,,,80,,
+AR46,,,,GI01SUMO-RII11-02-PCO2WC053,7,1,,,CGMGI-01SUMO-00007,,CGINS-PCO2WC-C0106,,,,130,,
+AR46,,,,GI01SUMO-RII11-02-PHSENE041,7,1,,,CGMGI-01SUMO-00007,,CGINS-PHSENE-P0171,,,,20,,
+AR46,,,,GI01SUMO-RII11-02-PHSENE042,7,1,,,CGMGI-01SUMO-00007,,CGINS-PHSENE-P0176,,,,100,,
+AR46,,,,GI01SUMO-SBC11-00-CPMENG000,7,1,,,CGMGI-01SUMO-00007,,CGCON-BCPM01-50010,,,,0,,
+AR46,,,,GI01SUMO-SBD11-00-DCLENG000,7,1,,,CGMGI-01SUMO-00007,,CGCON-BDCL11-50010,,,,0,,
+AR46,,,,GI01SUMO-SBD11-01-MOPAK0000,7,1,,,CGMGI-01SUMO-00007,,CGINS-MOPAK0-43258,,,,0,,
+AR46,,,,GI01SUMO-SBD11-03-HYDGN0000,7,1,,,CGMGI-01SUMO-00007,,CGINS-HYDGN0-20010,,,,0,,
+AR46,,,,GI01SUMO-SBD11-04-DOSTAD000,7,1,,,CGMGI-01SUMO-00007,,CGINS-DOSTAD-00211,,,,1,,
+AR46,,,,GI01SUMO-SBD11-05-SPKIRB000,7,1,,,CGMGI-01SUMO-00007,,CGINS-SPKIRB-00229,,,,-6,,
+AR46,,,,GI01SUMO-SBD11-06-METBKA000,7,1,,,CGMGI-01SUMO-00007,,CGINS-METLGR-00013,,,,0,,
+AR46,,,,GI01SUMO-SBD11-08-NUTNRB000,7,1,,,CGMGI-01SUMO-00007,,CGINS-NUTNRB-01094,,,,1,,
+AR46,,,,GI01SUMO-SBD12-00-DCLENG000,7,1,,,CGMGI-01SUMO-00007,,CGCON-BDCL12-50010,,,,0,,
+AR46,,,,GI01SUMO-SBD12-01-OPTAAD000,7,1,,,CGMGI-01SUMO-00007,,CGINS-OPTAAD-00209,,,,1,,
+AR46,,,,GI01SUMO-SBD12-02-FLORTD000,7,1,,,CGMGI-01SUMO-00007,,CGINS-FLORTD-01218,,,,1,,
+AR46,,,,GI01SUMO-SBD12-03-HYDGN0000,7,1,,,CGMGI-01SUMO-00007,,CGINS-HYDGN0-20009,,,,0,,
+AR46,,,,GI01SUMO-SBD12-04-PCO2AA000,7,1,,,CGMGI-01SUMO-00007,,CGINS-PCO2AA-35263,,,,0,,
+AR46,,,,GI01SUMO-SBD12-05-WAVSSA000,7,1,,,CGMGI-01SUMO-00007,,CGINS-WAVSSA-05981,,,,0,,
+AR46,,,,GI01SUMO-SBD12-06-METBKA000,7,1,,,CGMGI-01SUMO-00007,,CGINS-METLGR-00015,,,,0,,
+AR46,,,,GI01SUMO-SBD12-08-FDCHPA000,7,1,,,CGMGI-01SUMO-00007,,CGINS-FDCHPA-43401,,,,-6,,

```

Comment:
Reviewed CUID changes and deployment SNs for D00007. Should the NSIF CTDBP SN be 50142? That's what it is in the config sheet. Also, per discussion in the cg-data-team channel the HYD sns would be switched to the vendor SN.

Reply:
Looks good. I updated the CTDBP-F (NSIF) serial number to 50142

Reply:
reviewed JB update to CTDBP-F (NSIF) - looks good